### PR TITLE
Bump containerd to 1.0.2 (cfd04396dc68220d1cecbe686a6cc3aa5ce3667c)

### PR DESCRIPTION
**Pull request by thaJeztah**
Release notes: https://github.com/containerd/containerd/releases/tag/v1.0.2

Full diff: https://github.com/containerd/containerd/compare/9b55aab90508bd389d7654c4baf173a981477d55...cfd04396dc68220d1cecbe686a6cc3aa5ce3667c

Also bump dependencies:

- https://github.com/containerd/go-runc/compare/ed1cbe1fc31f5fb2359d3a54b6330d1a097858b7...4f6e87ae043f859a38255247b49c9abc262d002f
- https://github.com/containerd/cgroups/compare/29da22c6171a4316169f9205ab6c49f59b5b852f...c0710c92e8b3a44681d1321dcfd1360fc5c6c089
- https://github.com/stevvooe/ttrpc/compare/76e68349ad9ab4d03d764c713826d31216715e4f...d4528379866b0ce7e9d71f3eb96f0582fc374577
- containerd/continuity (already ahead)
- opencontainers/runc (already ahead)

### CODE REVIEW

**Review by AkihiroSuda — APPROVED** [review]
LGTM if green

**Comment by thaJeztah** [comment]
@dnephin I didn’t bump the censored package (as that one was picked from master); let me know if you think we should bump to a more recent commit

**Comment by thaJeztah** [comment]
censored -> vendored damn you autocorrect :joy:

**Review by dnephin — APPROVED** [review]
LGTM

**Comment by kolyshkin** [comment]
Is there any specific reason to use git commit hash (`cfd04396....`) rather than the tag (`v1.0.2`)? To me, using a tag if possible looks cleaner and also self-descriptive.

**Comment by dnephin** [comment]
We have run into an issue in the past where a project used a branch name that looked like a tag (v1.x.x), which got us into trouble when they removed the branch. However in this case we know this to be a tag, and we can trust the project to not remove the tag, so we should be fine to use the tag I think.

**Comment by thaJeztah** [comment]
Reason I switched to a commit;

- As @dnephin mentioned; a commit is fixed, a tag not (we also had vendors re-tagging 😞)
- Main reason in _this_ case is https://github.com/moby/moby/pull/35770

**Review by vdemeester — APPROVED** [review]
LGTM 🌵

File: vendor.conf [hunk 1]
```diff
@@ -107,12 +107,12 @@ google.golang.org/genproto d80a6e20e776b0b17a324d0ba1ab50a39c8e8944
 github.com/containerd/containerd 3fa104f843ec92328912e042b767d26825f202aa
```
Comment (tophj-ibm) [inline vendor.conf]: just wondering, why we don't keep this the same as the sha in binaires-commits?

Comment (dnephin) [reply vendor.conf]: We want to use the official tagged releases for the binary, but the latest code for the client library, so they will be different SHAs.

**Pull request merged**